oslo_cache/backends/redis.py

```diff
@@ -0,0 +1,45 @@
+from dogpile.cache.backends import redis as redis_backend
+from oslo_cache import core
+__all__ = [
+    'RedisSentinelTLSBackend'
+]
+class RedisSentinelTLSBackend(redis_backend.RedisSentinelBackend):
+    def __init__(self,arguments):
+        """
+        Wrapper class for dogpile.cache.redis_sentinel.
+        Currently only translates arguments for dogpile.cache.backends.RedisSentinelBackend
+        because user is unable to pass them using oslo config in it's current format
+        arguments:
+           sentinels: list of sentinels formated as sentinel1IP:port,sentinel2IP:port
+           ssl: turns on TLS
+           ssl-<keyfile,certfile,ca_certs>: TLS certificate, key and CA
+           username,password: login credentials 
```

Comment:
```suggestion
           username,password: login credentials
```

https://www.flake8rules.com/rules/W291.html